spyder/plugins/explorer/widgets/explorer.py

```diff
@@ -1185,7 +1179,8 @@ def delete_file(self, fname, multiple, yes_to_all):
         if yes_to_all is None:
             answer = QMessageBox.warning(
                 self, _("Delete"),
-                _("Do you really want to delete <b>%s</b>?"
+                _("Do you really want to delete <b>%s</b>?\n"
+                  "The file will be moved to the trash."
```

Comment:
```suggestion
                  "The file will be moved to the trash can."
```

Small improvement to this new text.